src/test/java/nextstep/session/domain/DimensionTest.java

```diff
@@ -10,15 +10,15 @@ class DimensionTest {
     @Test
     public void 이미지는_최소_너비와_높이를_충족해야_한다() {
         assertThatThrownBy(() -> {
-            new SessionCoverImage(100, 180, 10000, "png");
+            new SessionCoverImage(1L, 100, 180, 10000, "png");
         }).isInstanceOf(InvalidImageConditionsException.class)
             .hasMessageContaining("이미지는 최소 300*200 이상이여야합니다. 입력된 사이즈 100*180");
     }
 
     @Test
     public void 이미지는_너비와_높이가_3대2_이어야_한다() {
```

Comment:
```suggestion
    public void 이미지는_너비와_높이_비율이_존재한다() {
```
저는 좀 더 일반화된 이름을 사용할것 같아요.
왜냐하면, `3`,`2` 라는 값은 언제든지 변경될수 있기 때문에 수정범위를 좁히는데에 도움이 된다고 생각하기 때문이에요 😄